internal/page/fixtures/donor.go

```diff
@@ -303,6 +304,19 @@ func updateLPAProgress(
 		donorDetails.Tasks.PeopleToNotify = actor.TaskCompleted
 	}
 
+	if data.Progress == slices.Index(progressValues, "addCorrespondent") {
+		donorDetails.AddCorrespondent = form.Yes
+		donorDetails.Correspondent = makeCorrespondent(Name{
+			Firstnames: "Jonathan",
+			Lastname:   "Ashfurlong",
+		})
+
+		donorDetails.Tasks.AddCorrespondent = actor.TaskCompleted
+
+		donorDetails.CheckedAt = time.Time{}
+		donorDetails.Tasks.CheckYourLpa = actor.TaskNotStarted
```

Comment:
these should be the default values

Reply:
Ah yeah, side effect of the placement of this section in the progress list. I'll remove.